core/src/main/java/com/datastrato/gravitino/authorization/NameIdentifierUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.authorization;
+
+import com.datastrato.gravitino.Entity;
+import com.datastrato.gravitino.NameIdentifier;
+
+class NameIdentifierUtils {
+
+  private NameIdentifierUtils() {}
+
+  public static NameIdentifier ofUser(String metalake, String user) {
+    return NameIdentifier.of(
+        metalake, Entity.SYSTEM_CATALOG_RESERVED_NAME, Entity.USER_SCHEMA_NAME, user);
+  }
+
+  public static NameIdentifier ofGroup(String metalake, String group) {
+    return NameIdentifier.of(
+        metalake, Entity.SYSTEM_CATALOG_RESERVED_NAME, Entity.GROUP_SCHEMA_NAME, group);
+  }
+
+  public static NameIdentifier ofRole(String metalake, String role) {
+    return NameIdentifier.of(
+        metalake, Entity.SYSTEM_CATALOG_RESERVED_NAME, Entity.ROLE_SCHEMA_NAME, role);
+  }
+}
```

Comment:
You can add these methods in `AuthorizationUtils`, no need to use another class.